ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/dhcp-servers.yml

```diff
@@ -0,0 +1,34 @@
+dhcp_servers:
+  - vrf: TEST
+    ipv4_vendor_options:
+      - vendor_id: NTP
+        sub_options:
+          - code: 42
+            type: ipv4-address
+            data: "10.1.1.1"
+          - code: 1
+            type: string
+            data: "test"
```

Comment:
lets add the third type of option in the test

Reply:
Done.